scripts/deleteAndLoadSnapsets.js

```diff
@@ -0,0 +1,523 @@
+/*!
+Copyright 2018 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// This script modifies the preferences database:
+// 1. Update the views records for accessing Prefs Safes and GPII Keys,
+// 2. Retrieves all the Prefs Safes of type "snapset" (prefsSafesType = "snapset") from the databsse.
+// 3. Retrieves all the GPII Keys associated with each snapset Prefs Safe so found,
+// 4. Deletes these Prefs Safes and their associated GPII Keys from the database,
+// 5. Uploads the new Prefs Safes and their GPII Keys to the database,
+// A sample command that runs this script:
+// node deleteAndLoadSnapsets.js $COUCHDBURL $STATIC_DATA_DIR $BUILD_DATA_DIR
+
+"use strict";
+
+var http = require("http"),
+    url = require("url"),
+    fs = require("fs"),
+    fluid = require("infusion");
+
+var gpii = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.dataLoader");
+
+fluid.setLogging(fluid.logLevel.INFO);
+
+// Handle command line
+if (process.argv.length < 5) {
+    fluid.log("Usage: node deleteAndLoadSnapsets.js $COUCHDB_URL $STATIC_DATA_DIR $BUILD_DATA_DIR [--justDelete]");
```

Comment:
Nice to have `--justDelete` flag. Could you explain it in the code comment at the beginning of the script too? Thanks.